### PR TITLE
structure design 

### PR DESCRIPTION
**Pull request by GedeonTS**
## math-magician: structure design

In this pull request, I created :
- the project components folder
- and  the project UI using class-based components 

Your review is welcome! :smile:

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Calculator from "./components/Calculator";
+
+const App = () => {
+  return <Calculator />;
+};
+
```
Comment (ShadyShawkat) [inline src/App.jsx:7]: - [ ] Kindly make the App a class based component, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @GedeonTS, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.jsx (hunk 1 above)
Comment (GedeonTS) [reply src/App.jsx:7]: Hi @ShadyShawkat, thanks for your feedback! :+1:

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @GedeonTS , 

Great work on making the changes requested by the previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GedeonTS** [comment]
Hi @micheaol, thanks for approving my work!

**Pull request merged**